sdk/core/azure-core/src/main/java/com/azure/core/models/ResponseError.java

```diff
@@ -34,9 +29,7 @@ public final class ResponseError {
      * @param code the error code of this error.
      * @param message the error message of this error.
      */
-    @JsonCreator
-    public ResponseError(@JsonProperty(value = "code", required = true)String code,
-                         @JsonProperty(value = "message", required = true)String message) {
+    public ResponseError(String code, String message) {
```

Comment:
removing annotations would be a breaking change, are we ready to do it?

Reply:
Yes, it would be this is something we'll dive into further when this is ready to be merged back to main